_posts/2021-01-06-how-to-update-to-cis-13.adoc

```diff
@@ -0,0 +1,220 @@
+accidentally to a resource such as AWS Elastic Load Balancer (ELB), which can damage the credibility of the application/website behind the ELB. As a best practice, it is recommended to delete expired certificates. To help you
+achieve this recommendation, check out the https://github.com/gruntwork-io/terraform-aws-cis-service-catalog/tree/master/examples/cleanup-expired-certs/terraform[example] of the ready-made `cleanup-expired-certs` module.
+
+==== 2. Deploy IAM Access Analyzer module (recommendation 1.21)
+
+The updated recommendations also require that the AWS IAM Access Analyzer service is enabled across all active regions in a given AWS account or organization.
+
+Once enabled and active, this service will examine the trust policies and access to the following resources:
+
+- Amazon Simple Storage Service buckets;
+- AWS Identity and Access Management roles;
+- AWS Key Management Service keys;
+- AWS Lambda functions and layers;
+- Amazon Simple Queue Service queues.
+
+The IAM Access Analyzer will scan only within the AWS Account or Organization boundaries it has been enabled for. The results from this scan will be visible and accessible through the AWS CLI and the AWS Web console. For more information and details on what the AWS IAM Access Analyzer can achieve for your AWS Account and Organization, please refer to the official https://docs.aws.amazon.com/IAM/latest/UserGuide/what-is-access-analyzer.html[AWS docs].
+
+To help you achieve CIS 1.3 compliance, you can find examples of how to use the ready-made module as part of the `landingzone` module in the `terraform-aws-service-catalog` repository https://github.com/gruntwork-io/terraform-aws-service-catalog/releases/tag/v0.15.4[v0.15.4], and specifically the `account-baseline-root` and `account-baseline-security` https://github.com/gruntwork-io/terraform-aws-service-catalog/tree/v0.15.4/examples/for-learning-and-testing/landingzone[examples].
+
+==== 3. Deploy the ebs-encryption-multi-region module (recommendation 2.2.1)
+
+EC2 supports encryption at rest when using the Elastic Block Store (EBS) service. While disabled by default, forcing encryption when creating EBS volumes is supported. Encrypting data at rest reduces the likelihood that it is
+unintentionally exposed and can nullify the impact of disclosure if the encryption remains unbroken. Recommendation 2.2.1 specifies a manual process to encrypt EBS volumes using the AWS Console; however, Gruntwork has developed
+a module that configures volume encryption by default in all enabled regions. Check out the https://github.com/gruntwork-io/terraform-aws-security/tree/master/modules/ebs-encryption-multi-region[ebs-encryption-multi-region] to
+configure AWS EBS encryption in all allowed regions of an AWS Account.
+
+==== 4. Deploy the vpc-app-network-acl and vpc-mgmt-network-acl modules (recommendation 5.1)
+
+If you are already using the `vpc-app-network-acl` and/or `vpc-mgmt-network-acl` modules, you need to update the
```

Comment:
NIT: add a sentence of context here about how we added a new `vpc-app-network-acl` module to the CIS repo, how this module enforces rules around SSH/RDP access, etc, and how customers will need to deploy this new module. _Then_ you can go into the part about what to do if you are using the `vpc-app-network-acl` module from `terraform-aws-vpc` already.